package.json

```
@@ -219,6 +219,21 @@
     "yaserver": "^0.4.0"
   },
   "overrides": {
+    "ts-api-utils": {
+      "typescript": "$typescript"
+    },
+    "@typescript-eslint/typescript-estree": {
+      "typescript": "$typescript"
+    },
+    "@typescript-eslint/utils": {
+      "typescript": "$typescript"
+    },
+    "ts-node": {
+      "typescript": "$typescript"
+    },
+    "tsec": {
+      "typescript": "^5.9.0-dev.20250613"
```

For some reason, with the current `package-lock.json` in place using `$typescript` here results in:

>Unable to resolve reference $typescript

However if you run `npm i` once with this config and update the lockfile, this can be switched to the same pattern moving forward.

```suggestion
      "typescript": "$typescript"
```